debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresErrorHandler.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql;
+
+import java.net.SocketException;
+
+import io.debezium.connector.base.ChangeEventQueue;
+import io.debezium.pipeline.ErrorHandler;
+import io.debezium.util.Throwables;
+
+/**
+ * Error handler for Postgres.
+ *
+ * @author Gunnar Morling
+ */
+public class PostgresErrorHandler extends ErrorHandler {
+
+    public PostgresErrorHandler(String logicalName, ChangeEventQueue<?> queue, Runnable onThrowable) {
+        super(PostgresConnector.class, logicalName, queue, onThrowable);
+    }
+
+    @Override
+    protected boolean isRetriable(Throwable throwable) {
+        Throwable rootCause = Throwables.getRootCause(throwable);
+
+        if (rootCause instanceof SocketException && rootCause.getMessage().contains("Broken pipe")) {
```

Comment:
I think there are more exceptions that can be retriable. 
E.g. the exception mentioned in the comments on https://issues.redhat.com/browse/DBZ-1325 does not contain "Broken pipe" as cause.

Reply:
Yes, good point. I'll add this one. I'd like to stay on the conservative end initially (we always can expand), but this one should be there.

Reply:
@gunnarmorling Maybe going to the root cause is too specific?
`Caused by: org.postgresql.util.PSQLException: Database connection failed when writing to copy` looks like a pretty safe default

Reply:
Reworked as per the suggestion.